.github/workflows/regsync-config.yaml

```diff
@@ -0,0 +1,36 @@
+# Generate-Regsync-Config action will checkout release-v2.7 branch, run make regsync target and
+# creates a pull request from rancherbot/charts or rancher/charts release-v2.7 branch with any image additions
+# to regsync config file. This action is triggered whenever something is pushed into release-v2.7 branch.
+
+name: Generate-Regsync-Config
+
+on:
+  push:
+    branches:
+      - release-v2.7
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: release-v2.7
+
+      # Need to remove export version once rancher/charts gets the latest version 
+      # of charts-build-script binary.
+      - name: Generate Regsync
+        run: |
+          export CHARTS_BUILD_SCRIPT_VERSION=v0.4.2 
+          make pull-scripts
+          make regsync
+      
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{secrets.PUSH_TO_FORKS_SUBMIT_PRS}}
+          push-to-fork: rancherbot/charts
+          title: 'Update regsync config images file'
+          branch-suffix: timestamp
+
```

Comment:
` reviewers: rohitsakala,nicholasSUSE, diogoasouza`
 
 If you want to set us always as reviewers automatically